### ISSUE TITLE
Multi-line function calls wrongly formatted

### ISSUE THREAD

**Issue by BalestraPatrick** [nicklockwood/SwiftFormat#495]
SwiftFormat is formatting this code:
```swift
lazy var builder = Builder<String>(
    param1: param1,
    param2: param2
)
    .action1 { print($0) }
    .action2 { print($0) }
    .action3 { reason in
        print(reason)
    }
```
into:
```swift
lazy var builder = Builder<String>(
    param1: param1,
    param2: param2
)
.action1 { print($0) }
.action2 { print($0) }
.action3 { reason in
    print(reason)
}
```

The first formatting is the preferred behavior by Xcode (ctrl+I), so I'd like to keep that formatting. What's the rule that's formatting the code in this way? I couldn't find it in the docs and I'd like to disable it for my project if not possible to configure to follow Xcode's behavior.

Thank you!

**Comment by lordzsolt** [comment]
The reason is because the default behavior by XCode formatting through Ctrl+I is simply horrible, and wrong.

In the top section, `action1, action2, action3` are on the same level as `param1, param2`, so if you quickly run your eye through the code, you could assume they are arguments passed to `Builder<String>.init`.

Here's another example that has been driving me crazy for years:

When you hit Ctrl+I on this block, nothing changes. The code is readable, the two closures are indented to the same level.
```
searchService.getSearchResults()
            .subscribe(onNext: { response in
                print("Success")
            }, onError: { error in
                print("Error")
            }).disposed(by: disposeBag)
```

But if you add `[weak self]`, Xcode suddenly loses it and formats it to

```
searchService.getSearch(parameters: parameters)
            .subscribe(onNext: { [weak self] response in
                print("Success")
                }, onError: { [weak self] error in
                    print("Error")
            }).disposed(by: disposeBag)
```

Which is clearly wrong. Running your eye though this code, you would assume `subscribe()` is taking a single closure, `onNext`, which is 3 lines long.

**Comment by nicklockwood** [comment]
@BalestraPatrick this is handled by the `indent` rule, which you can disable with `--disable indent`.

It's not currently possible to change this behavior without disabling all indenting functionality, but it would make sense for the `--xcodeindentation` option to affect this.

**Comment by nicklockwood** [comment]
@BalestraPatrick fixed in 0.46.0.
